source/develop/release-management/features/virt/virt-v2v-integration.html.md

```diff
@@ -139,6 +139,33 @@ Import from external environments using virt-v2v in terms of the general flow:
 
 10. Unlock VM & disks
 
+##### Import OVA from VMware
+
+Currently oVirt support importing OVA only from VMware hypervisor.
```

Comment:
actually we already know that we manage to work with OVA produced by AWS as well, right? let's put it differently: in principle, we supports only OVA files produced by VMware. However. this may work with OVA files produced by other vendors